client/src/App.jsx

```diff
@@ -1,59 +1,205 @@
 
```

Comment:
Awesome work on variable naming, useful and concise comments and code structure!